specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/dynatrace.json

```diff
@@ -0,0 +1,2244 @@
+            "schema": {
+              "$ref": "#/definitions/DynatraceSingleSignOnResourceListResult"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "tags": [
+          "SingleSignOn"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "SingleSignOn_List_MaximumSet_Gen": {
+            "$ref": "./examples/SingleSignOn_List_MaximumSet_Gen.json"
+          },
+          "SingleSignOn_List_MinimumSet_Gen": {
+            "$ref": "./examples/SingleSignOn_List_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Dynatrace.Observability/monitors/{monitorName}/listHosts": {
+      "post": {
+        "operationId": "Monitors_ListHosts",
+        "summary": "List the VM/VMSS resources currently being monitored by the Dynatrace resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/VMHostsListResponse"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "tags": [
+          "Monitors"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Monitors_ListHosts_MaximumSet_Gen": {
+            "$ref": "./examples/Monitors_ListHosts_MaximumSet_Gen.json"
+          },
+          "Monitors_ListHosts_MinimumSet_Gen": {
+            "$ref": "./examples/Monitors_ListHosts_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Dynatrace.Observability/monitors/{monitorName}/getMetricStatus": {
```

Comment:
>/getMetricStatus" [](http://example.com/codeflow?start=128&length=17)

Just want to point out that creating such post actions does not work well with ARM echo system like ARM templates. Cant we model this as ARM resource?

Reply:
We have previous post actions like this and hence following the same format. If such changes are required, then we would like to do at all places. Can we take it as a separate task in the future.

Reply:
That would be a breaking change too. Please the breaking change policy here https://aka.ms/AzBreakingChangesPolicy/

Reply:
We already got approval while merging to private repo: https://github.com/Azure/azure-rest-api-specs-pr/pull/12287